lib/nexthop.h

```diff
@@ -139,6 +140,13 @@ struct nexthop {
 
 	/* SR-TE color used for matching SR-TE policies */
 	uint32_t srte_color;
+
+	/* SRv6 localsid info for Endpoint-behaviour */
+	enum seg6local_action_t nh_seg6local_action;
+	struct seg6local_context *nh_seg6local_ctx;
+
+	/* SRv6 Headend-behaviour */
+	struct in6_addr *nh_seg6_segs;
```

Comment:
@mjstapp 

> I'm still uncomfortable with adding these three fields to the nexthop struct. I think this should be a single 'nh_srv6' block, a single pointer, instead of three independent values that occupy more space.

On following commit, I updated property of `struct nexthop`. I added newly nh_srv6 property and it is including seg6local-action, seg6local-ctx and seg6-segs. 

https://github.com/FRRouting/frr/pull/5865/commits/424e54a55c9fc72e94e55beb3b26a8b377b59e6d
https://github.com/FRRouting/frr/pull/5865/commits/adf402a807d0545effbcebc48ebd80b1f74f0201
https://github.com/FRRouting/frr/pull/5865/commits/7694bc765ff9384b842776e32709803e9d33e77d